google/cloud/bigtable/internal/rpc_policy_defaults.h

```diff
@@ -0,0 +1,114 @@
+// AUTOGENERATED BY tools/generate_rpc_policy_defaults.py; DO NOT
+// EDIT MANUALLY. ALL MANUAL CHANGES WILL BE OVERWRITTEN.
+//
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGTABLE_INTERNAL_RPC_POLICY_DEFAULTS_H_
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGTABLE_INTERNAL_RPC_POLICY_DEFAULTS_H_
+
+#include "google/cloud/bigtable/version.h"
+#include <chrono>
+
+namespace google {
+namespace cloud {
+namespace bigtable {
+inline namespace BIGTABLE_CLIENT_NS {
+namespace internal {
+
+struct RPCPolicyDefaults {
```

Comment:
I think this class represents the `RPCPolicyParameters`, they are not the defaults because we no longer have defaults for the policy (we have default policy parameters for each class).

Reply:
renamed to RPCPolicyParameters.